dali/test/python/test_dali_tf_dataset_shape.py

```diff
@@ -0,0 +1,191 @@
+import tensorflow as tf
+import nvidia.dali.ops as ops
+import nvidia.dali.pipeline as pipeline
+import nvidia.dali.plugin.tf as dali_tf
+import nvidia.dali.types as dali_types
+
+import numpy as np
+
+import os
+from nose.tools import assert_equals, raises
+
+data_path = os.path.join(os.environ['DALI_EXTRA_PATH'], 'db/single/jpeg/')
+file_list_path = os.path.join(data_path, 'image_list.txt')
+
+def dali_pipe_batch_1(shapes, types, as_single_tuple = False):
+    target_sizes = [(853, 1280, 3),
```

Comment:
I don't like this strong bound between the content of DALI extra and test itself.
I would run a standalone DALI pipeline next to it and get the actual dimension from it and then compare with one returned by DALIDataset.
Otherwise any change to 'db/single/jpeg/' will break this test.

Reply:
Done.